libr/core/cmd_section.c

```diff
@@ -232,10 +264,10 @@ static int cmd_section(void *data, const char *input) {
 		}
 		break;
 	case '\0':
-		r_io_section_list (core->io, core->offset, 0);
+		print_sections (core->bin, core->offset, R_FALSE);
```

Comment:
I think r_io_section_list have been used because you want ```S``` to display only what it's really mapped, but the problem is that sections are not what you really map. For this reason I've chosen to just print the sections.

Do you think it's better to keep r_io_section_list? If  you keep it you get something like:
```
[00] * 0x00000000 r-x va=0x00400000 sz=0x06dc vsz=0x200000 phdr0
[01] . 0x000006e0 rw- va=0x006006e0 sz=0x0230 vsz=0x200000 phdr1
[02] * 0x00000000 rwx va=0x00000000 sz=0x21da vsz=0x21da ehdr
```

because in ELF (but probably also in MACH0) you map program headers, not sections.